Habitica/src/main/java/com/habitrpg/android/habitica/ui/fragments/social/GuildsOverviewFragment.java

```diff
@@ -32,6 +35,9 @@
     @BindView(R.id.chat_refresh_layout)
     SwipeRefreshLayout swipeRefreshLayout;
 
+    @Inject
+    ChallengeLocalRepository challengeLocalRepository;
```

Comment:
The local repositories should not be used directly, but the methods should be made available through the regular repository.